data-es2016plus.js

```diff
@@ -6498,7 +6498,7 @@ exports.tests = [
         res: {
```

Comment:
The `actual` output in Node 16.0 is
```
[
  'global',
  'ignoreCase',
  'multiline',
  'dotAll',
  'unicode',
  'sticky',
  'hasIndices'
]
```
Yet `/a/dgimsuy.flags` does give `"dgimsuy"`. It seems to me `hasIndices` for some reason is called after other flags but the return value is somehow reordered. Has this issue been reported to Chromium?

Reply:
Yes, it's an open bug. I added the test case specifically because https://npmjs.com/regexp.prototype.flags is needed to patch it in latest node/chrome.